omnigibson/macros.py

```diff
@@ -68,6 +68,9 @@
 # (Demo-purpose) Whether to activate Assistive Grasping mode for Cloth (it's handled differently from RigidBody)
 gm.AG_CLOTH = False
 
+# Forced light intensity for all USD-loaded objects
```

Comment:
seems like also None is supported? can we also mention that here in the comment?